configs/approval.json

```diff
@@ -1,133 +1,162 @@
 {
   "roles": [
     {
-      "name": "Approval User",
-      "description": "An approval user role which grants permissions to read/create a request.",
+      "name": "Approval Administrator",
       "system": true,
-      "platform_default": true,
-      "version": 2,
+      "version": 4,
```

Comment:
@hsong-rh Should this be version 3 and not 4.

Reply:
Do we need to keep `version` in sync?